specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-05-01-preview/migrate.json

```diff
@@ -12188,6 +12209,64 @@
         }
       }
     },
+    "AvsRecommendedNode": {
```

Comment:
Do not use recommended Node here.

Reply:
The word "Recommended" is being used in the response wide in the platform. We are in sync with it